linode/manager
# M3-5483: Add "Upgrade to NVMe" chip for volumes pending migration

## Description

- Add `volume_migration_scheduled` and `volume_migration_imminent` to the `NotificationType` interface
- Add "Upgrade to NVMe" chip to volumes that have a "volume_migration_pending" notification
- Clicking the chip will redirect the page to the attached linode's detail page
- Update mocks

## How to test

Turn on MSW and go to `/volumes` and you should see a volume with the "Upgrade to NVMe" chip
